libs/wallet/src/connectors/vega-connector.ts

```diff
@@ -449,7 +449,22 @@ export type CreateReferralSet = {
   };
 };
 
+export enum MarginMode {
```

Comment:
Typically enum-ish types have been defined as union types. Does the wallet not accept strings of 'MARGIN_MODE_CROSS_MARGIN and MARGIN_MODE_ISOLATED_MARGIN ?

Reply:
wallet accepts 1 or 2